lib/vm-run

```diff
@@ -109,7 +109,7 @@ vm::run(){
     fi
 
     # default bhyve options
-    _opts="-AHP"
+    _opts="-AH"
```

Comment:
Afaik, -P is not intended for debugging only:
```
     -P          Force the guest virtual CPU to exit when a PAUSE instruction
                 is detected.
```

Reply:
I'm pretty sure it's the reason why my VMs randomly terminate instead of rebooting.